examples/rules.kts

```diff
@@ -179,6 +179,25 @@ val ruleSet = ruleSet(ortResult, licenseInfoResolver) {
         }
     }
 
+    packageRule("VULNERABILITY_IN_PACKAGE") {
+        val maxAcceptedSeverity = "5.0"
+        val scoringSystem = "CVSS2"
+
+        require {
+            -isExcluded()
+            +hasVulnerabilityWithSeverityOver(maxAcceptedSeverity, scoringSystem) { o1, o2 ->
+                o1.toFloat().compareTo(o2.toFloat())
+            }
+        }
+
+        issue(
+            Severity.ERROR,
+            "Vulnerability with $scoringSystem severity > $maxAcceptedSeverity in package " +
```

Comment:
Broken English - I would make it "The package ${pkg.id.toCoordinates()} has a vulnerability with $scoringSystem severity > $maxAcceptedSeverity" so it aligns with the other rule messages.